index.html

```diff
@@ -40,6 +40,187 @@
+      </section>
+
+      <section>
+        <h2>Watering</h2>
+        <p>Orchids need to be watered once the potting medium has dried completely. Once dry, pour in room tempurature water to soak the plant.</p>
+        <p>Orchids are very susceptible to <a href=#>root rot</a>, so it is important not to over water them.</p>
+        <p>You don't need to use an ice cube. I would reccomend against it.</p>
+      </section>
+
+      <section>
+        <h2>Lighting</h2>
+        <p>Orchids require bright, indirect sunlight to grow healthily. Direct sunlight can burn the leaves, causing your orchid to shrivel and die.</p>
+      </section>
+
+      <section>
+        <h2>Climate</h2>
+        <p>Orchids are tropical plants by nature, meaning they do best in warm, humid environments. If you live in a dry area, you can simulate this by misting the orchid with room tempurature water, avoiding the delicate flowers.</p>
+      </section>
+
+      <section>
+        <h3>Feedback</h3>
+        <p>Questions about plant care? Concerns about the content shown on the page? Any other coment?</p>
+        <p>Please fill out the form below, and I will be in touch as soon as possible.</p>
+      </section>
+
+      <section>
+        <h3>Form</h3>
+        <form>
+          <fieldset>
+            <legend>Information</legend>
+            <Label for="name">Name</Label>
+            <input type="text" name="name" id="name" placeholder="Jane Doe" autocomplete="name" required>
+            <label for="email">Email</label>
+            <input type="email" name="email" id="email" placeholder="janedoe@email.com" autocomplete="email" required>
+          </fieldset>
+          <fieldset>
+            <legend>Feedback</legend>
+             <label>Feedback type:</label>
+            <input type=radio name="Feedback" id="question" value="question" checked>
+             <label for="question">Question</label>
+            <input type="radio" name="Feedback" id="comment" value="comment">
+              <label for="comment">Comment</label>
+            <input type="radio" name="Feedback" id="concern" value="concern">
+              <label for="concern">Concern</label><br>
+
+            <label for="text">Write your feedback here:</label><br>
+              <textarea name="text" id="text" rows="10" cols="40" required></textarea>
+          </fieldset>
+          <fieldset>
+            <legend>Consent</legend>
+            <input type="checkbox" id="marketing" name="marketing">
+              <label for="marketing">I consent to receiving marketing emails</label><br>
+            <input type="checkbox" id="terms" name="terms" required>
+              <label for="terms">I have read and agreed to the <a href=#>Terms and Conditions</a></label>
+          </fieldset>
+          <input type="submit">
+        </form>
+      </section>
+
+    </main>
+
+    <footer>
+      <img src="2291219-200.png" alt="logo of a potted vine">
+      <h4>Happy Plant Gardening</h4>
```

Comment:
Your headings should start at an h2 in your <header> and <footer>

Your code is showing an h4.